pipeline/talend_test/harvest_results/6-nec-hob.emii.org.au/soop_tmv/soop_tmv/soop_tmv_nrt_trajectory_map.csv

```diff
@@ -0,0 +1,3 @@
+file_id,url,size,time_coverage_start,time_coverage_end,CPHL_b,PSAL_b,TEMP_b,TURB_b,geom,platform_code,vessel_name
+1,IMOS/SOOP/SOOP-TMV/VLST_Spirit-of-Tasmania-1/realtime/transect/1sec/2019/IMOS_SOOP-TMV_TSUB_20190313T085121Z_VLST_FV00_transect-M2D_END-20190313T184130Z.nc,2010968,2019-03-13 08:51:21,2019-03-13 18:41:30,t,t,t,t,0102000020E61000000F000000C1CAA145B61D62405B423EE8D9EC42C0CC5D4BC8071D624044FAEDEBC02943C04182E2C7981B62408A8EE4F21F2A43C07958A835CD1562404CA60A46252543C07FD93D79581462404BC8073D9B2543C060E5D022DB136240FD87F4DBD72943C06C787AA52C236240F54A5986388E43C02EFF21FDF625624068B3EA73B5A543C05917B7D10036624024B9FC87F40344C04ED1915CFE3F6240D5E76A2BF64744C094F6065F984262407D3F355EBA5144C0933A014D84436240B22E6EA3016444C0E6AE25E483466240C286A757CA6A44C0E02D90A0F84B6240174850FC189344C08126C286A74B624058CA32C4B19644C0,VLST,Spirit-of-Tasmania-1
+3,IMOS/SOOP/SOOP-TMV/VLST_Spirit-of-Tasmania-1/realtime/transect/1sec/2019/IMOS_SOOP-TMV_TSB_20081207T085601Z_VLST_FV02_transect-M2D_END-20081207T194931Z.nc,2010968,2019-03-13 08:51:21,2019-03-13 18:41:30,t,t,t,t,0102000020E61000000F000000C1CAA145B61D62405B423EE8D9EC42C0CC5D4BC8071D624044FAEDEBC02943C04182E2C7981B62408A8EE4F21F2A43C07958A835CD1562404CA60A46252543C07FD93D79581462404BC8073D9B2543C060E5D022DB136240FD87F4DBD72943C06C787AA52C236240F54A5986388E43C02EFF21FDF625624068B3EA73B5A543C05917B7D10036624024B9FC87F40344C04ED1915CFE3F6240D5E76A2BF64744C094F6065F984262407D3F355EBA5144C0933A014D84436240B22E6EA3016444C0E6AE25E483466240C286A757CA6A44C0E02D90A0F84B6240174850FC189344C08126C286A74B624058CA32C4B19644C0,VLST,Spirit-of-Tasmania-1
```

Comment:
I don't understand the last line. The file is from 2008, but is in a 2019 folder with bad time_coverage_start and end.  I checked the inside of the file, and it's completely different

Reply:
Yeah,sorry - should have added a comment.  I couldn't find any FV02 files which passed the compliance checker, so I renamed a couple of FV00 files to test the FV02 processing.

Reply:
If you have some good FV02 files I can update the test to use them instead

Reply:
@jonescc unfortunately, the last FV02 file is from 2014, so I'm afraid there aren't any